SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestMultipleOwnership/SpatialTestMultipleOwnership.cpp

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestMultipleOwnership.h"
+#include "MultipleOwnershipPawn.h"
+#include "SpatialFunctionalTestFlowController.h"
+
+#include "GameFramework/PlayerController.h"
+#include "Kismet/GameplayStatics.h"
+
+/**
+ * This test automates the MultipleOwnesrhip gym which demonstrates that RPCs can be sent from multiple actors that have their owner set to
+ * a player controller. The test contains 1 Server and 2 Clients.
```

Comment:
I see that the test should not rely on anything specific in the map.
I would add it to a generated map so that it's run in CI - either the SpatialNetworkingMap, although it kind of duplicates parts of the possession test. Or a new CI_Nightly map, which would be basically the same as the SpatialNetworkingMap but for running nightly.

Hmm, I think SpatialNetworkingMap should be ok.....
Also, did you try running this test in native networking? I think it should pass, but just to make sure (tests in the spatialnetworkingmap are run both in native and spatial networking)